src/app/public/testing/country-field/country-field-fixture.ts

```diff
@@ -0,0 +1,168 @@
+import {
+  ComponentFixture,
+  tick
+} from '@angular/core/testing';
+
+import {
+  DebugElement
+} from '@angular/core';
+
+import {
+  By
+} from '@angular/platform-browser';
+
+import {
+  SkyAppTestUtility
+} from '@skyux-sdk/testing';
+
+import {
+  SkyCountryFieldCountry
+} from '@skyux/lookup';
+
+/**
+ * Allows interaction with a SKY UX country field component.
+ */
+export class SkyCountryFieldFixture {
+  private debugEl: DebugElement;
+  private countries: SkyCountryFieldCountry[];
+
+  constructor(
+    private fixture: ComponentFixture<any>,
+    skyTestId: string
+  ) {
+    this.debugEl = SkyAppTestUtility.getDebugElementByTestId(fixture, skyTestId, 'sky-country-field');
+
+    this.countries = JSON.parse(
+      JSON.stringify((window as any)
+        .intlTelInputGlobals
+        .getCountryData()
+      )
+    );
+  }
+
+  /**
+   * The value of the input field's autocomplete attribute.
+   */
+  public get autocomplete(): string {
```

Comment:
Could we rename this `autocompleteAttribute` to match the component's property name? We have an autocomplete component and it comes with some predefined meanings of what "autocomplete" means, so we explicitly name this `autocompleteAttribute` to convey it controls the actual HTML attribute and not a SKYUX component.